inlong-audit/audit-service/src/main/java/org/apache/inlong/audit/config/OpenApiConstants.java

```diff
@@ -52,12 +54,14 @@ public class OpenApiConstants {
     public static final String AUDIT_TAG = "auditTag";
     public static final String INLONG_GROUP_Id = "inlongGroupId";
     public static final String INLONG_STREAM_Id = "inlongStreamId";
+    public static final String IP = "ip";
+    public static final String AUDIT_CYCLE = "auditCycle";
```

Comment:
```suggestion
    public static final String AUDIT_CYCLE = "audit.cycle";
```

Reply:
The auditCycle is an OpenAPI parameter, the name has been optimized.